CoreWidgetProvider/Widgets/SSHWalletWidget.cs

```diff
@@ -324,13 +353,33 @@ public override void UpdateWidget()
         WidgetManager.GetDefault().UpdateWidget(updateOptions);
     }
 
+    public void UpdateWidget(string patternHost)
```

Comment:
I think it's a little confusing to have this UpdateWidget which is kind of like the overridden UpdateWidget but also not. Can use the regular UpdateWidget(), and get the patternHost into the data in the GetData(WidgetPageState page) method?

Reply:
Fair point! I'll look into it!